homeassistant/components/light/plum_lightpad.py

```diff
@@ -0,0 +1,102 @@
+"""
+Support for Plum Lightpad switches.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/light.plum_lightpad
+"""
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.light import (
+    PLATFORM_SCHEMA, ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS, Light)
+
+REQUIREMENTS = ['plumlightpad==0.0.5']
+
+CONF_USER = 'username'
+CONF_PASSWORD = 'password'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_USER, default=None): cv.string,
+    vol.Optional(CONF_PASSWORD, default=None): cv.string,
+})
+
+
+def setup_platform(hass, config, add_devices_callback, discovery_info=None):
+    """Setup the demo light platform."""
+    from plumlightpad import Plum
+
+    if (config.get(CONF_USER) is not None and
```

Comment:
If you don't want them to be None, mark them as `Required` in `PLATFORM_SCHEMA` and remove the `default`.